.npmignore

```diff
@@ -14,3 +14,4 @@ _vimrc_local.vim
 *.prefs
 ./.npmignore
 coverage
+sample-code/javascript-wd/node_modules
```

Comment:
This should just be `sample-code`. There is no need to distribute any of the sample code with the package.